### DISCUSSION TITLE
fontsizeinput - Syncronize with the normal font-size selection box / fix format

### DISCUSSION THREAD

**Discussion by wolfomat** [tinymce/tinymce#9198] [General]
Hi there,

on the Version 6.4 of tinyMCE i've added the plugin:
    fontsizeinput

and i've defined:
`    font_size_input_default_unit: 'pt',`

Somehow, the editor shows up like this:
![tiny-mce-font](https://github.com/tinymce/tinymce/assets/17848544/bf178d96-973a-476b-91f5-1cdd3cc23ae1)

So, when i change the font size, the values are synchronized. but it still shows px instead of pt.
Furthermore, the steps are always 1 Unit.

How can i defined 10pt as well for the fontsizeinput field and how can i modify the steps? maybe 0.5 units instead of 1 unit.

i've consulted the documentation, but i did not stumple accross any information.

thanks.

was set aswell.

**Comment by lostkeys** [comment]
Hey @wolfomat.

These two buttons was never really meant to be used together like this. The "right" answer is that you should pick the one that best fits your use-case.

Curious to know more why you want to use both at the same time.

— Fredrik, Product Manager

**Reply by wolfomat** [reply]
Hey @lostkeys , thank you very much for the quick reply.

In my personal opinion i wanted to have this feature a bit "word-like".
However, we've decided to remove the drop down list. Sadly, the input still shows 'px' instead of PTs
`  font_size_input_default_unit: 'pt',`

Is there a default value for the size? maybe that's the problem.

**Reply by wolfomat** [reply]
Hi @lostkeys ,

it seems to be a bug. i've filed a bug report: 

https://github.com/tinymce/tinymce/issues/9216
